app/src/main/java/com/example/trainingintive/dog_images_feature/presentation/AlbumFragment.kt

```diff
@@ -15,13 +17,33 @@ class AlbumFragment : Fragment() {
     @Inject
     lateinit var viewModelFactory: ViewModelFactory
 
+    @Inject
+    lateinit var adapter: AlbumAdapter
+
     val albumViewModel: AlbumViewModel by lazy {
         ViewModelProvider(this, viewModelFactory)[AlbumViewModel::class.java]
     }
 
+    override fun onAttach(context: Context) {
+        (requireActivity().application as MyApplication).appComponent.inject(this)
+        super.onAttach(context)
+    }
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View = inflater.inflate(R.layout.fragment_album, container, false)
+    ): View {
+        val binding = FragmentAlbumBinding.inflate(inflater, container, false)
+        binding.lifecycleOwner = viewLifecycleOwner
+        val recyclerView = binding.recyclerView
```

Comment:
1. Please stick to some order convention, declare `vars` firstly, then make some operations, don't mix the order.
2. It can be simplified, like this:
```
binding.apply {
    lifecycleOwner = viewLifecycleOwner
    recyclerView.adapter = adapter
}
```